### DISCUSSION TITLE
Bluetooth Microphone

### DISCUSSION THREAD

**Discussion by ierotheo** [blueman-project/blueman#2182] [Q&A]
Has anybody succeed to have a Bluetooth microphone working?
I was not able to have one working in Linux Mint DE 5 and now on LMDE 6 I am having the same issue.

**Comment by ierotheo** [comment]
I am running blueman-manager 2.3.5

**Comment by infirit** [comment]
You'll need to change to one of the headset profiles, HSP/HFP.
